src/components/header/Header.tsx

```diff
@@ -90,15 +93,52 @@ const Header = () => {
       console.log(modalData);
     });
   };
-  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
-  const openMenu = anchorEl;
+  const [menuAnchorEl, setMenuAnchorEl] = React.useState<null | HTMLElement>(
+    null
+  );
+
+  const openMenu = menuAnchorEl;
   const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
-    setAnchorEl(event.currentTarget);
+    setMenuAnchorEl(event.currentTarget);
   };
+
   const handleClose = () => {
-    setAnchorEl(null);
+    setMenuAnchorEl(null);
+    setOpen(false);
   };
 
+  const BASE_URL = "https://www.dealbuddy.co.nz/api";
```

Comment:
Why hard code URL?

Reply:
Sir i remove base url and create .env file for BASE_URL and store it into .env file.